File: java/Class1.java
public class Class1 {
    private String privateVar;

    public Class1() {
        this.privateVar = "I'm private";
    }

    public void method1() {
        System.out.println("This is method1 from Class1");
        privateMethod();
    }

    private void privateMethod() {
        System.out.println("This is a private method in Class1");
        System.out.println("Private var: " + privateVar);
    }
}

File: php/Class1.php
<?php
class Class1 {
    private $privateVar;
    public static $staticVar = "I'm a static variable in Class1";
    
    public function __construct() {
        $this->privateVar = "I'm private";
    }
    
    public function method1() {
        echo "This is method1 from Class1\n";
        $this->privateMethod();
    }
    
    private function privateMethod() {
        echo "This is a private method in Class1\n";
        echo "Private var: " . $this->privateVar . "\n";
        echo "Static var: " . self::$staticVar . "\n";
    }
}